### ISSUE TITLE
Move Arm commands to Arm subsystem

### ISSUE THREAD

**Issue by nlaverdure** [TripleHelixProgramming/Crescendo#49]
Use the [Instance Factory Method](https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#instance-command-factory-methods) design pattern:

```
public Arm() {
...
public Command lowerArmCommand() {
return this.startEnd(() -> this.pneumaticRetract(), () -> {});
}
}
```

Then call this command as needed in RobotContainer. Examples:

`new JoystickButton(m_operator, Button.kLeftBumper.value).onTrue(m_arm.lowerArmCommand);`

```
public void teleopInit() {
m_arm.lowerArmCommand().schedule();
}
```

**Comment by jasenj1** [comment]
I pushed a change to the lower-arm-on-teleopinit branch to implement this.

**Comment by nlaverdure** [comment]
Closed with #46
